lessons/codap-transformers2/langs/en-us/index.adoc

```diff
@@ -0,0 +1,81 @@
+= More Transformers: Sort
+@description{Students learn about filltering  - manually, deleting one row at a time, or by using a _Transformer_ to build and modify a copy of the table.}
+
+[@lesson-intro-table]
+|===
+@lesson-prereqs{codap-bar-charts}
+| Lesson Goals
+| Students will be able to...
+
+* write a contract, purpose statement, and expression
+* use the `Sort` Transformer order the Animals Dataset by a number of criteria
+
+| Student-facing Lesson Goals
+|
+
+* Let’s learn how transform one table into another.
+
+| Materials
+|[.materials-links]
+
+@material-links
+
+|===
+
+== Sorting @duration{25 minutes}
+
+=== Overview
+Students _change a table,_ sorting it by a particular column. Then, students learn a new and more dynamic way of sorting... with the Sort Transformer!
+
+=== Launch
+
+[.lesson-instruction]
+- Open the @starter-file{animals} in CODAP.
+- How many animals' names start with the letter S?
+** _Once you notice students deep in concentration, looking for and tallying animals with S-names, move on to the next question..._
+- What would make it easier to figure out how many animals' names start with the letter S?
+
+Explain to students that, in this lesson, they will learn two ways of sorting. The first way is to actually alter the original table. The second way is to create a transformed copy of the original table, which automatically has the animals sorted.
+
+[.lesson-instruction]
+- When manipulating data, what do you think might be some advantages and disadvantages to changing the actual table?
+** _Advantage: It's quick, easy and intuitive._
+** _Disadvantage: If I want to refer to the previous arrangement of animals for any reason... it might no longer exist._
+- What are some advantages and disadvantages to creating a new modified table?
+** _Advantage: I will be able to refer back to my old table, and maybe even compare the new table with the old table. I can use my Transformer again later once I save it!_
+** _Disadvantage: It will take a bit of typing to set it up, and a few extra clicks._
+
+You will direct students to sort various columns on the table by clicking the attribute name and then selecting from the drop-down menu either `Sort Ascending` or `Sort Descending`. This will modify the original table.
+
+[.lesson-instruction]
+- Let's sort the animals alphabetically by name and see if we get the same answer as we did before! _(The image below shows where to click to sort the animals alphabetically by name.)_
```

Comment:
alphabetically = ascending, yes? This is why you might need to define ascending and descending, above